### DISCUSSION TITLE
Is Forestry.io headless CMS Admin not working with new trailing slash redirects?

### DISCUSSION THREAD

**Discussion by devklepacki** [vercel/next.js#16416] [Help]
I have always accessed Forestry's localy deployed admin panel within route `/admin`.
The route was serving a static _index.html_ file, which was appending some stuff to the url, so it looked like this `/admin/index.html/#/login`.

However right now `index.html/` probably gets redirected to slash-less form, so the url ends up as `/admin/index.html#/login` thus not working and falling into redirects loop.

This may be due to the new rules:
```
┌ source: /:file((?:[^/]+/)*[^/]+\.\w+)/
├ destination: /:file
```
and
```
┌ source: /:path+/
├ destination: /:path+
```
I am not sure how this was even working before.
I've found some _not-that-old_ similar thread on vercel github, however I'm not sure if it's any usefull here: https://github.com/vercel/docs/issues/563

For now I am unable to access localy deployed Forestry CMS Admin.
How do we get over this? Am I missing anything?

**Comment by SimonBarak** [comment]
By default Next.js will redirect urls with trailing slashes to their counterpart without a trailing slash. You need to set trailingSlash in **next.config.js**

`
  module.exports = {
    trailingSlash: true,
  }
`
https://nextjs.org/docs/api-reference/next.config.js/trailing-slash
